apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/server/RandomFissClaimGenerator.java

```diff
@@ -134,29 +133,22 @@ public class RandomFissClaimGenerator extends AbstractRandomClaimGenerator<FissC
   private static final List<FissNonBillRevCode> FissNonBillRevCodeEnums =
       enumValues(FissNonBillRevCode.values());
 
-  /**
-   * Creates an instance with the specified seed.
-   *
-   * @param seed seed for the PRNG
-   */
-  public RandomFissClaimGenerator(long seed) {
-    super(seed, false, Clock.systemUTC());
-  }
+  /** Max length of a FISS claim id. */
+  private static final int FissClaimIdLength = 32;
```

Comment:
Is this a kotlin naming convention?

Reply:
Nothing new there.  It's far more readable and keeps the names shorter so google format doesn't insert so many line breaks.